rousseau_vote/lib/src/screens/success_screen.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:rousseau_vote/src/l10n/rousseau_localizations.dart';
+import 'package:rousseau_vote/src/screens/polls_screen.dart';
+import 'package:rousseau_vote/src/util/ui_util.dart';
+import 'package:rousseau_vote/src/widgets/rounded_button.dart';
+import 'package:rousseau_vote/src/widgets/rousseau_app_bar.dart';
+import 'package:rousseau_vote/src/widgets/rousseau_logged_scaffold.dart';
+
+class SuccessScreen extends StatelessWidget{
+  SuccessScreen({this.message});
+  String message;
+  static const String ROUTE_NAME = '/success';
+
+  @override Widget build(BuildContext context) {
+    // TODO: implement build
+      return Scaffold(
```

Comment:
Use RousseauLoggedScaffold instead. This ensure that the user is logged in and if not it redirects to the login screen.